README.md

```diff
@@ -40,29 +40,33 @@ Show all routes for a bus line
 * docker / docker-compose
 
 ### How to run
+Put an .osm.pbf file into docker/imposm/import directory.
+
 Run `docker-compose up -d` to run the services in detached mode.
 
-The first run will perform several actions :
+The first run will perform several actions:
 
 * download all the required base images,
 * build and run the containers
 * create the database
 * import the osm.pbf file in the `docker/imposm/import/` directory
 * execute the post_process SQL file
 
+On first time you may need to run it twice before it works.
```

Comment:
I think this issue is covered by the workaround bellow, so there is no need to this.